### DISCUSSION TITLE
Automating New Project Creation in Vercel via API Endpoints

### DISCUSSION THREAD

**Discussion by nishiurahiroki** [vercel/community#4626] [Feature Question]
### Question

I am looking to automate the process of creating new projects in Vercel, not through the UI, but by utilizing API endpoints. While I am aware that there are API endpoints available for deploying new versions of existing projects, it seems like there aren’t any API endpoints provided for creating the projects themselves. Is automation of this process possible?

**Comment by amyegan** [comment]
You can create a new project using the `/v9/projects` endpoint. 😄 

https://vercel.com/docs/rest-api/endpoints#create-a-new-project

**Comment by nishiurahiroki** [comment]
@amyegan 
Thank you for your quick response! It was really helpful. I'll reach out again if I have more questions🫶
